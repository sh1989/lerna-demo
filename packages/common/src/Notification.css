.notification {
  border: 1px solid red;
  background: lightcoral;
  padding: 20px;
}
