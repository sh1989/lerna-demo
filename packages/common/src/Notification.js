import React from 'react';
import style from './Notification.css';

const Notification = ({ msg }) => (
  <div className={style.notification}>
    { msg }
  </div>
);

export default Notification;
