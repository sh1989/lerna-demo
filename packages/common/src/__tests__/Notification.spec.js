import React from 'react';
import Notification from '../Notification';
import ShallowRenderer from 'react-test-renderer/shallow';

test('Notification Component', () => {
    const renderer = new ShallowRenderer();
    renderer.render(
        <Notification msg="A message" />
    );
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
});