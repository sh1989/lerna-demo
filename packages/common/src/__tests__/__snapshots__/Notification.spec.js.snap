// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Component 1`] = `
<div
  className="notification"
>
  A message
</div>
`;
